.github/workflows/pull_request.workflow.yml

```diff
@@ -0,0 +1,114 @@
+          node-version: "12"
+      - uses: ./.github/actions/es-lint
+
+  unit-tests:
+     name: Unit Tests
+     runs-on: ubuntu-latest
+     steps:
+       - uses: actions/checkout@v2
+       - name: Cache node modules
+         uses: actions/cache@v2
+         env:
+           cache-name: cache-node-modules
+         with:
+           path: ~/.npm
+           key: ${{ runner.os }}-build-${{ env.cache-name }}-${{ hashFiles('**/package-lock.json') }}
+           restore-keys: |
+             ${{ runner.os }}-build-${{ env.cache-name }}-
+             ${{ runner.os }}-build-
+             ${{ runner.os }}-
+       - uses: actions/setup-node@v1.4.4
+         with:
+           node-version: "12"
+       - uses: ./.github/actions/unit-tests
+
+  integration-tests:
+     name: Integration Tests
+     runs-on: ubuntu-latest
+     steps:
+       - uses: actions/checkout@v2
+       - name: Cache node modules
+         uses: actions/cache@v2
+         env:
+           cache-name: cache-node-modules
+         with:
+           path: ~/.npm
+           key: ${{ runner.os }}-build-${{ env.cache-name }}-${{ hashFiles('**/package-lock.json') }}
+           restore-keys: |
+             ${{ runner.os }}-build-${{ env.cache-name }}-
+             ${{ runner.os }}-build-
+             ${{ runner.os }}-
+       - uses: actions/setup-node@v1.4.4
+         with:
+           node-version: "12"
+       - uses: ./.github/actions/integration-tests
+         env:
+          CYPRESS_RECORD_KEY_DOC: ${{ secrets.CYPRESS_RECORD_KEY_DOC }}
+
+  docs-tests:
+     name: Documentation Snippet Tests
+     runs-on: ubuntu-latest
+     steps:
+       - uses: actions/checkout@v2
+       - name: Cache node modules
+         uses: actions/cache@v2
+         env:
+           cache-name: cache-node-modules
+         with:
+           path: ~/.npm
+           key: ${{ runner.os }}-build-${{ env.cache-name }}-${{ hashFiles('**/package-lock.json') }}
+           restore-keys: |
+             ${{ runner.os }}-build-${{ env.cache-name }}-
+             ${{ runner.os }}-build-
+             ${{ runner.os }}-
+       - uses: actions/setup-node@v1.4.4
+         with:
+           node-version: "12"
+       - uses: ./.github/actions/snippet-tests
+         env:
+          CYPRESS_RECORD_KEY_DOC: ${{ secrets.CYPRESS_RECORD_KEY_DOC }}
+
+  dead-links:
+    name: Dead links
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Cache node modules
+        uses: actions/cache@v2
+        env:
+          cache-name: cache-node-modules
+        with:
+          path: ~/.npm
+          key: ${{ runner.os }}-build-${{ env.cache-name }}-${{ hashFiles('**/package-lock.json') }}
+          restore-keys: |
+            ${{ runner.os }}-build-${{ env.cache-name }}-
+            ${{ runner.os }}-build-
+            ${{ runner.os }}-
+      - uses: actions/setup-node@v1.4.4
+        with:
+          node-version: "12"
+      - uses: ./.github/actions/dead-links
+
```

Comment:
We don't test the admin console when opening a pull request on `7-dev` ?

Reply:
We do, all theses jobs run when you open a PR

Reply:
I don't see the Admin Console action on this workflow (but I can see it on the [pull_request_master](https://github.com/kuzzleio/sdk-javascript/pull/568/files#diff-a7c4e17a25267febc5552e0693d0c2d5b76e42811fb80a848cee1bf804c71dc9R9) workflow)